packages/sanity/src/core/releases/hooks/usePerspective.tsx

```diff
@@ -147,36 +148,38 @@ export function usePerspective(): PerspectiveValue {
     (perspectiveId: string) =>
       Boolean(
         excludedPerspectives?.includes(
-          isPublishedPerspective(perspectiveId) ? 'published' : `bundle.${perspectiveId}`,
+          isPublishedPerspective(perspectiveId) ? 'published' : perspectiveId,
         ),
       ),
     [excludedPerspectives],
   )
 
   return useMemo(
     () => ({
-      perspective,
+      selectedPerspectiveName: currentPerspectiveName,
+      selectedReleaseName:
+        currentPerspectiveName === 'published' ? undefined : currentPerspectiveName,
       excludedPerspectives,
       setPerspective,
       setPerspectiveFromReleaseDocumentId: setPerspectiveFromReleaseDocumentId,
       setPerspectiveFromReleaseId: setPerspectiveFromReleaseId,
       toggleExcludedPerspective,
-      currentGlobalBundle,
-      currentGlobalBundleId: isPublishedPerspective(currentGlobalBundle)
+      selectedPerspective: currentPerspective,
+      globalReleaseDocumentId: isPublishedPerspective(currentPerspective)
```

Comment:
does this needs to be `globalReleaseDocumentId` ? 
feels better: `selectedPerspectiveId`